src/game_interpreter_battle.cpp

```diff
@@ -37,6 +37,18 @@ enum BranchBattleSubcommand {
 	eOptionBranchBattleElse = 1
 };
 
+enum TargetType {
+	Actor,
+	Member,
+	Enemy,
+};
+
+static const char* target_text[] = { "actor", "party member", "enemy" };
+
+static const void MissingTargetWarning(char* command_name, TargetType target_type, int target_id) {
```

Comment:
``command_name`` must be ``const char*`` to get rid of the compiler warning. After pushing best is to wait for the Linux runners to finish and look for warnings in the diff. They are usually more pedantic than Visual Studio 😅

Reply:
Good tip, thanks!